dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/TestHideMessages.java

```diff
@@ -56,6 +56,14 @@ public void setupTest() throws Exception {
         msgServer.startServer();
     }
 
+    @After
+    public void cleanUp() throws Exception {
+        if (msgServer != null && msgServer.isStarted()) {
+            msgServer.stopServer("CWMCG0007E", "CWMCG0014E", "CWMCG0015E",
+                                 "CWMCG5003E", "CWPMI2006W", "CWMMC0013E", "CWWKG0033W");
```

Comment:
Are these Warning/Error messages related to the test cases? It seems like some are from MP Metrics...